api/envoy/config/trace/v2/trace.proto

```diff
@@ -46,8 +45,8 @@ message Tracing {
     // - :ref:`DynamicOtConfig <envoy_api_msg_config.trace.v2.DynamicOtConfig>`
     // - :ref:`DatadogConfig <envoy_api_msg_config.trace.v2.DatadogConfig>`
     // - :ref:`OpenCensusConfig <envoy_api_msg_config.trace.v2.OpenCensusConfig>`
-    // [#comment: TODO(marco) when XRay is implemented, uncomment the following; - :ref:`XRayConfig
-    // <envoy_api_msg_config.trace.v2.XRayConfig>`]
+    // [#comment: TODO(marcomagdy) when XRay is implemented, uncomment the following; - :ref:`XRay
```

Comment:
I think we can call this support done now so let's uncomment this. We can do future PRs as enhancements.